packages/rum/src/domain/record/observer.ts

```diff
@@ -182,7 +194,7 @@ export function initInputObserver(cb: InputCallback): ListenerHandler {
 
     let inputState: InputState
     if (type === 'radio' || type === 'checkbox') {
-      if (nodePrivacyLevel === NodePrivacyLevel.MASK) {
+      if (shouldMaskNode(target, nodePrivacyLevel)) {
```

Comment:
I don't really inderstand why we need `shouldMaskNode()`, if we already have `getNodePrivacyLevel()`
There is probably a confusion between the **Mask** term in `shouldMaskNode()` and the `NodePrivacyLevel.MASK`